pkger/parser.go

```diff
@@ -844,6 +845,10 @@ func (r Resource) intShort(key string) int {
 	return i
 }
 
+func (r Resource) int32Short(key string) int32 {
```

Comment:
lets leave out all int32 stuffs from teh Resource. Leave it as as int, and convert it at when we assign the value in the inflxudb.ViewProperty type. I don't see it being used anyhow 🤷‍♂